app/src/main/java/org/grammatek/simacorrect/spellcheckerservice/YfirlesturAnnotation.kt

```diff
@@ -157,7 +157,9 @@ class YfirlesturAnnotation(
             for (r in results) {
                 val annotations = arrayListOf<Annotations>()
                 for (annotation in r.annotations!!) {
-                    annotations.add(annotation)
+                    if (!(annotation.code == "Z002" && annotation.start == 0)) {
```

Comment:
Use a comment for this magic string or even better: make a constant with an appropriate name ?